tests/SpecTests/FunctionalTestCase.php

```diff
@@ -301,10 +293,7 @@ private function isServerlessRequirementSatisfied(?string $serverlessMode): bool
     /**
      * Checks if server version and topology requirements are satifised.
      *
-     * @param string|null $minServerVersion
-     * @param string|null $maxServerVersion
-     * @param array|null  $topologies
-     * @return boolean
+     * @param array|null $topologies
```

Comment:
Seems redundant given `?array $topologies = null` below.

Reply:
Removed this and a number of other superfluous type annotations.